src/script.rs

```diff
@@ -1147,7 +1147,7 @@ mod test {
         assert_eq!(v_nonmin_alt.unwrap(), slop_v_nonmin_alt.unwrap());
     }
 
-	#[test]
+        #[test]
```

Comment:
indent

Reply:
This is bugging me because locally I don't have this change D:

Reply:
Make sure you are diffing against current master. Maybe another rebase helps!